Source/Examples/Example01_windows_blit_tree/Main.cpp

```diff
@@ -0,0 +1,205 @@
+#include "Application.h"
+#include "FrameBuffer.h"
+#include "Timer.h"
+#include <iostream> 
+#include <math.h>
+#include <thread>
+
+#define PI 3.14159265
```

Comment:
```suggestion
constexpr double PI = 3.14159265;
```
It will also replace PI in the period of compiling. The advantage is that constexpr can have type, it is more safe. And it doesn't need to care about () for some macro cases.

Reply:
Sure, I will change it as the commit.